### DISCUSSION TITLE
Add users programatically with an API in a tina project

### DISCUSSION THREAD

**Discussion by webtwitter** [tinacms/tinacms#3564] [Ideas]
Is there a way (or will there be a way) to add users from my nextjs site directly in my tina project?
So not only manual...

**Comment by scottgallant** [comment]
A few other people have asked for this.  Any more details on why this would be helpful for you?

**Reply by webtwitter** [reply]
Yes, I'm seeing this helpful because I'd like to create a portal where people create their own website, so when they register they will be added automatically as new users. Might be also awesome to create collections with api too, but I think there might be workarounds already for this. So I think it might also serve like an auth API.
